### DISCUSSION TITLE
Welcome to Simple-IDS Discussions!

### DISCUSSION THREAD

**Discussion by jasonish** [jasonish/simple-ids#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gordnhoo** [comment]
Is there a way to edit the suricata.yaml i tried it but when i restarted it, it created new docker containers changing my config

**Reply by jasonish** [reply]
Not, but it should. Ideally there should be menu entries for the most common changes.

What edit were you wanting to make?

**Reply by gordnhoo** [reply]
Hi thx for the reply  l want to only get alerts from the eve-log so i want to remove everything and only leave the alerts because im going to give the json to a SIEM

**Reply by jasonish** [reply]
Ah, I'm not sure if Simple-IDS is the best for that.  It kind of wants to own everything.. But 2 features I'll be looking to add are:
- if ~/.config/simple-ids/suricata.yaml exists, use it (or something like that)
- menus to disable/enable various eve types

**Reply by gordnhoo** [reply]
ah ok i found this tool very useful for a quick setup to test some rules since it was very easy to deploy,will be looking for the next update.Also i saw you a lot in the suircata forums you helped me solve some problems already. Thanks a lot
